CoMD/src-threaded/ljForce.cpp

```diff
@@ -166,14 +170,21 @@ int ljForce(SimFlat* s)
    const real_t eShift = POT_SHIFT * rCut6 * (rCut6 - 1.0);
 
    {
-     RAJA::kernel<ljForcePolicy>(
+#ifdef DO_CUDA
```

Comment:
I'd rather not have the ifdef here - when you define the policies, can you name them the same name?